share/templates/CMakeLists.txt

```diff
@@ -17,6 +17,10 @@
 #  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 #=============================================================================
 
+# adding or renaming templates here should also better be done in convert.json, 
+# but definitly need to get reflected in ../instruments/instrumentsxml.h,
```

Comment:
Typo in `definitly` ;-)

Reply:
fixed

Reply:
How many typos can a single person to in a single PR of this small size ;-(